plugin/trino-hive/src/main/java/io/trino/plugin/hive/fs/TrinoFileStatus.java

```diff
@@ -31,6 +32,7 @@
 {
     private final List<BlockLocation> blockLocations;
     private final Path path;
+    private final Location location; // Duplicate of path, needed during TrinoFileSystem migration
```

Comment:
I replaced `Path` with `Location` in https://github.com/trinodb/trino/pull/17190/commits/77f10960937a146d92f88f75c924e3335ee999e9 which is part of #17190